package.json

```diff
@@ -38,7 +38,7 @@
     "@govuk-react/table": "^0.9.1",
     "@govuk-react/text-area": "^0.9.1",
     "@govuk-react/visually-hidden": "^0.9.1",
-    "@ministryofjustice/frontend": "^0.2.6",
+    "@ministryofjustice/frontend": "^4.0.2",
```

Comment:
I've not braved the upgrade to v5.x since that was looking a bit more involved then I was prepared to do.